tezos-encoding/src/nom.rs

```diff
@@ -584,8 +584,7 @@ where
 {
     move |input| {
         let (rest, result) = parser(input)?;
-        let hash = crypto::blake2b::digest_256(&input[..input.len() - rest.len()])
-            .map_err(|e| nom::Err::Failure(NomError::hash_error(input, e)))?;
```

Comment:
does this make `hash_error` unused too?

Reply:
Good catch! Removed in 7935bacb3e6bf24e562fb1bf24cb548c3652ad1b